daily_milestone.go

```diff
@@ -101,20 +148,25 @@ func createMilestoneData(advance int) []string {
 }
 
 func createMilestones(baseURL string, token string, projectID string, milestones []string) string {
-	strurl := []string{"https://", baseURL, "/projects", projectID, "/milestones"}
+	strurl := []string{"https://", baseURL, "/projects/", projectID, "/milestones"}
 	url := strings.Join(strurl, ",")
-
+	client := &http.Client{}
 	for _, m := range milestones {
 		req, err := http.NewRequest("POST", url, nil)
 		if err != nil {
 			logger.Println(err)
 			break
 		}
 		req.Header.Add("", m)
-		req.Header.Add("Private-Token", token)
-
+		req.Header.Add("PRIVATE-TOKEN", token)
+		resp, err := client.Do(req)
+		if err != nil {
+			logger.Println(err)
+			break
+		}
+		defer resp.Body.Close()
 	}
-	return "Milestones Created" + strings.Join(milestones, ",")
+	return ("Milestones Created" + strings.Join(milestones, ","))
```

Comment:
I think you prepared, but didn't finish your thought of using fmt.Printf().

Reply:
yes It didn't work the way I tried will try it next time.